DOCS.md

```diff
@@ -71,6 +71,38 @@ Generally speaking, you should AVOID defining `husky` in multiple `package.json`
 }
 ```
 
+Alternatively, you can use the `root` property to tell `husky` to only install if the [git `toplevel`](https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt---show-toplevel) directory is the same as the one with the installing `package.json`:
+
+```sh
+.
+└── root # git toplevel points here
+    ├── .git
+    ├── package.json 🐶 # husky will be installed as a depenceny *and* hooks will be installed
```

Comment:
spelling: dependency
\+ in a few other places below

Reply:
So it seems IntelliJ doesn't spell check markdown sh code blocks?

Thanks for catching this!